test/RegexTests.py

```diff
@@ -0,0 +1,50 @@
+import unittest
+
+import sys, os.path
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), '../src/asm2cfg'))
+
+import asm2cfg
```

Comment:
Yes, I dislike this part as well and I'd very much prefer you variant. But when I use it I get
```
$ python3 -m unittest test.RegexTests
E
======================================================================
ERROR: RegexTests (unittest.loader._FailedTest)
----------------------------------------------------------------------
ImportError: Failed to import test module: RegexTests
Traceback (most recent call last):
  File "/usr/lib/python3.8/unittest/loader.py", line 154, in loadTestsFromName
    module = __import__(module_name)
  File "/home/yugr/src/asm2cfg/test/RegexTests.py", line 3, in <module>
    from asm2cfg import asm2cfg
ModuleNotFoundError: No module named 'asm2cfg'
```
I couldn't find a better solution but it may be my lack of knowledge of how imports work...

Reply:
Done.